tests/test_physics/test_gaussian/test_symplectics.py

```diff
@@ -34,6 +40,10 @@
 from mrmustard.lab.states import TMSV, Thermal, Vacuum
 from mrmustard.physics.gaussian import controlled_X, controlled_Z
 
+from mrmustard import settings
+
+settings.HBAR = 2
```

Comment:
I guess it's for the same reason, but why do we need to set hbar here?

Reply:
Thanks for spotting this, I had forgotten it there